app.js

```diff
@@ -247,6 +247,28 @@ app.post('/inventory/remove_item', function (req, res) {
     })
 })
 
+// Inventory - Modify Item Route
+app.post('/inventory/modify_item', function(req, res) {
+    
+    // Grab the necessary data from the POST request body
+    var id = req.body.modal_shelf_id;
+    var min = req.body.modal_shelf_minimum_threshold
+    var max = req.body.modal_shelf_maximum_threshold
+    
+    // Form the SQL Query needed to update the shelf thresholds
+    var mod_inventory_query_string = "UPDATE products SET shelf_min_threshold = " +
+    min + ", shelf_max_threshold = " + max + " WHERE id = " + id
```

Comment:
```suggestion
    var mod_inventory_query_string = `UPDATE products 
                                                              SET shelf_min_threshold='${min}', shelf_max_threshold='${max}'
                                                              WHERE id='${id}'`
```
Same functionality, but maybe a little easier to read.